src/commands/economy/CiPay.ts

```diff
@@ -0,0 +1,47 @@
+import { Message, GuildMember } from "discord.js";
+import { Command } from "discord-akairo";
+import { CiEmbed } from "@structures";
+import { Collection } from "discord.js";
+import { Snowflake } from "discord.js";
+
+export default class SendCommand extends Command {
+  constructor() {
+    super("начислить", {
+      aliases: ["начислить", "Pay"],
+      args: [
+        {
+          index: 0,
+          id: "count",
+          type: "number"
+        },
+        {
+          index: 1,
+          id: "receiver",
+          type: "string"
+        },
+        {
+          index: 2,
+          id: "receiver",
+          match: "content",
+          type: "CiMembers"
+        }
+      ]
+    });
+  }
+
+  async exec(
+    message: Message,

```

Comment:
`async exec(mesasage: Message) `

can deconstruct to 

`async exec({ member, content, channel, guild }: Message) `